### ISSUE TITLE
Analysis fails for any public repo I've tried

### ISSUE THREAD

**Issue by ComanAlexandru** [shobrook/adrenaline#103]
I've tried for the following repos:
https://github.com/axelor/axelor-open-platform
https://github.com/axelor/axelor-open-suite
https://github.com/frappe/erpnext

It downloads the repo, maps it but then in fails for Code Analysis, Data Flow Analysis and Generate sugg. questions.
Also I don't get any email about the failures.

I've already paid for it and it failed at first try, would appreciate some guidance.

**Comment by rudrabhoj** [comment]
Did it work/got resolved?

**Comment by ComanAlexandru** [comment]
No, I've canceled the subscription. Looks like the project is not maintained but still accepting payments...

**Comment by shobrook** [comment]
Hmm this should have been resolved a while ago. Please try it again and re-open this issue if you have problems. Thank you!

**Comment by ComanAlexandru** [comment]
Can you give me free credits to try? Last time I've paid it didn't work and I won't pay again to try
